load-tests/cache/deployment/final.yaml

```diff
@@ -0,0 +1,191 @@
+apiVersion: v1
```

Comment:
This is an auto-generated file through `kustomize` if I recall correctly. Not supposed to be committed.